radiography/src/main/java/radiography/FocusedWindowViewFilter.kt

```diff
@@ -0,0 +1,13 @@
+package radiography
+
+import android.view.View
+
+/**
+ * Filters out root views that don't currently have the window focus from the output of [Radiography.scan].
+ */
+class FocusedWindowViewFilter : ViewFilter {
+
+  override fun matches(view: View): Boolean {
+    return view.parent != null || view.hasWindowFocus()
```

Comment:
need to double check that `view.parent != null` actually works here, we need a sample module with an android app.

also we should have UI tests ;)

Reply:
Separate PR?

Reply:
yeah. if it turns out it didn't work I'll fix it.